sklearn/utils/_plotting.py

```diff
@@ -100,3 +100,39 @@ def _interval_max_min_ratio(data):
     """
     diff = np.diff(np.sort(data))
     return diff.max() / diff.min()
+
+
+def _validate_style_kwargs(default_style_kwargs, user_style_kwargs):
+    """Create valid style kwargs by avoiding matplotlib aliases error.
+
+    Matplotlib raises an error when, for example, ‘color’ and ‘c’,
+    or 'linestyle' and 'ls', are specified. To avoid this,
+    we automatically keep only the one specified by the user
+    and we raise an error if the user specifies both.
+    """
```

Comment:
```suggestion
    """Create valid style kwargs by avoiding Matplotlib alias errors.

    Matplotlib raises an error when, for example, 'color' and 'c', or 'linestyle' and
    'ls', are specified together. To avoid this, we automatically keep only the one
    specified by the user and raise an error if the user specifies both.

    Parameters
    ----------
    default_style_kwargs : dict
        The Matplotlib style kwargs used by default in the scikit-learn display.
    user_style_kwargs : dict
        The user-defined Matplotlib style kwargs.

    Returns
    -------
    valid_style_kwargs : dict
        The validated style kwargs taking into account both default and user-defined
        Matplotlib style kwargs.
    """
```